app/services/force/event_subscriber_translate_service.rb

```diff
@@ -0,0 +1,80 @@
+require 'restforce'
+require 'faye'
+
+module Force
+  class EventSubscriberTranslateService
+    Event = Struct.new(:listing_id, :last_modified_date, :entity_name, :changed_fields,
+                       :replay_id, :updated_values, keyword_init: true)
+
+    def initialize
+      setup_salesforce_client
+      setup_faye_client
+    end
+
+    def listen_and_process_events
+      EM.run do
```

Comment:
This will run indefinitely if there are no exceptions encountered. We need to add some logic for ending this process, restarting this process, and handling exceptions so that process will keep going.